Polyhedron/demo/Polyhedron/Plugins/Mesh_3/C3t3_io_plugin.cpp

```diff
@@ -448,7 +449,55 @@ try_load_other_binary_format(std::istream& is, C3t3& c3t3)
     Fake_c3t3::Triangulation,
     C3t3::Triangulation,
     Update_vertex,
-    Update_cell>(is, c3t3.triangulation());
+    Update_cell>(is, c3t3.triangulation(),
+                 Update_vertex(),
+                 Update_cell()
+#ifdef CGAL_CXX11
+                 ,
+                 /* call_back: a C++11 lambda */
+                 [](std::istream& is,
+                    const C3t3::Triangulation::Triangulation_data_structure& tds,
+                    const char* text = 0)
+                 {
+                   static CGAL::Real_timer timer;
+                   static double last_time = 0;
+
+                   if(!timer.is_running()) {
+                     std::cerr << "Start loading...\n";
+                     timer.start();
+                   }
+                   if(text != 0) { std::cerr << text; return; }
+                   const double current_time = timer.time();
+                   if(current_time > last_time + 1.) {
+                     std::cerr << current_time << ": ";
+                     std::cerr << "Loaded " << tds.number_of_vertices()
+                               << " vertices, " << tds.number_of_cells()
+                               << " cells";
+                     typedef std::istream::pos_type pos_type;
+                     static pos_type last_pos = 0;
+                     const pos_type curr = is.tellg();
+                     is.seekg(0, std::ios_base::end);
+                     const pos_type end = is.tellg();
+                     is.seekg(curr);
+                     const std::streamoff divider =
+                       end > 500000000 ? 1000000 : 1000;
+                     const char unit =
+                       end > 500000000 ? 'M' : 'k';
+                     if(curr != pos_type(-1) && end != pos_type(-1)) {
+                       std::cerr << " (" << (curr/divider) << " "
+                                 << unit << "B/"
+                                 << (end/divider) << " "
+                                 << unit << "B, "
+                                 << ((curr-last_pos) / divider)
+                                 << unit << "B/s)\n";
+                     } else std::cerr << "\n";
+                     last_time = current_time;
+                     last_pos = curr;
+                   }
+                 }
+#endif // CGAL_CXX11
+                 );
+  std::cerr << "DONE.\n";
```

Comment:
For the moment, the callback is this big C++11 lambda function, that:
  - uses a timer to return very fastly but every second,
  - and displays stats about:
    - the size of the triangulation being read,
    - the size of the portion of the file that has been read, and the full size of the file.

Optionally, this callback can get a string as argument, so that the function force the display of a given string.